calculators/candidate_supporting_expenditures.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# Calculate independent expenditures in support of a candidate.
+class CandidateSupportingExpenditure
+  def initialize(candidates: [], ballot_measures: [], committees: [])
+    @committees = committees
+    @candidates = candidates
+  end
+
+  def fetch
+    # Get the total indepedent expenditures for candidates by date.
+    expenditures = ActiveRecord::Base.connection.execute(<<-SQL)
+      SELECT
+        "FPPC" AS "Filer_ID",
+        "Filer_NamL",
+        SUM("Amount") AS total
+      FROM "496"
```

Comment:
You need to include the schedule E-Expenditure data here.

Reply:
It was kind of strange-- I would have expected to see E-Expenditure data for years past, but didn't really see any. These school board races were in 2016 but the IE data was still in 496. So I'm not sure that the data ends up eventually on E-Expenditure. But nonetheless, I'll go in and add it.

Reply:
The independent expenditures are in Schedule D. They are also reported in Schedule E - but not by the candidate and/or ballot measure. Since the IE doesn't go to the campaign committee, the payee is usually a vendor.

Reply:
If the IE is made by a state or county general purpose committee or IE committee, they would generally only be required to file a 496 with Oakland, but not a 460 or 461.

Reply:
we are looking in Sched E for the IE committees but don't se the 496 data there.  Shouldn't appear there at the next deadline?

Reply:
You won't necessarily see a line item in Schedule E matching each transaction listed on the 496, because the spending is being reported differently. But you will see a transaction listed in Schedule D matching each transaction listed on a 496 during that reporting period. Does that make sense?

Reply:
Got it. So I need to include schedule D-Expenditure data in here as well, if I want to capture the totality of the independent expenditures supporting a certain candidate.